### PR TITLE
Add experimental flag

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

File: https/kingpinflag/flag.go [hunk 1]
```diff
@@ -20,7 +20,16 @@ import (
 // To use the default Kingpin application, call AddFlags(kingpin.CommandLine)
 func AddFlags(a *kingpin.Application) *string {
 	return a.Flag(
-		"web.config",
-		"Path to config yaml file that can enable TLS or authentication.",
+		"web.config.file",
```
Comment (SuperQ) [inline https/kingpinflag/flag.go:23]: Do we want to change this? I don't want to break existing node_exporter users by changing the flag name.

Comment (roidelapluie) [reply https/kingpinflag/flag.go:23]: #21 will break node_exporter users :(

Note exporter can keep its current flag as hidden.

Comment (SuperQ) [reply https/kingpinflag/flag.go:23]: Yea, good point, we can keep a compatibility flag alias.

**Review by SuperQ — APPROVED** [review]

**Comment by roidelapluie** [comment]
Actually we should make it experimental for everyone, and not have a dedicated function I think.

**Comment by roidelapluie** [comment]
I even think that http.config.file would be better

**Review by SuperQ — APPROVED** [review]
LGTM

**Pull request merged**